Sources/SpeziAccount/Views/AccountOverview/AccountOverviewSections.swift

```diff
@@ -138,7 +139,11 @@ struct AccountOverviewSections: View {
         sectionsView
             .injectEnvironmentObjects(service: service, model: model, focusState: $focusedDataEntry)
             .animation(nil, value: editMode?.wrappedValue)
-
+        
+        Section {
+            additionalContent
+        }
```

Comment:
Thanks for the PR, that looks really good. 🚀 

I just have a small comment, assuming this PR already represents the final state.
I don't think we should force `additionalContent` to be placed inside a single section. I think it would make sense to allow the user to define their own sections (including section headers and footers).

I have the remove/logout button separate it might make sense to wrap that into its own `Section`.

Reply:
Totally agree!